client/src/components/App.jsx

```diff
@@ -9,18 +10,22 @@ class App extends React.Component {
     this.state = {
       locationRatings: {},
       reviews: []
-    }
+    };
   }
 
-
   componentDidMount() {
     $.get(`http://localhost:1010/reviews/initial/${this.props.locationID}`)
```

Comment:
You'll want to refactor to get this port out of here. Any time you have http://localhost:1010/ , please replace with '/' instead. You want only relative URLS to local resources. This would break when running in a standard production environment.